.github/workflows/manifests.yml

```diff
@@ -0,0 +1,24 @@
+name: manifests
+
+on: [push, pull_request]
+
+jobs:
+  check:
+
+    runs-on: ubuntu-latest
+
+    env:
+      PYTHON_VERSION: 3.7
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python ${{ env.PYTHON_VERSION }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ env.PYTHON_VERSION }}
+      - name: Install Pipenv and Dependencies
+        run: |
+          python -m pip install --upgrade pipenv wheel
+      - name: OpenSearch 1.1
+        run: |
+          ./bundle-workflow/ci.sh manifests/opensearch-1.1.0.yml
```

Comment:
Rather than only run on the "current" manifest, could we have this enforced on all manifest files?

Reply:
Good idea.

Reply:
I figured out how to dynamically list manifest files and parallel run checks on them :)

<img width="883" alt="Screen Shot 2021-09-03 at 6 28 28 PM" src="https://user-images.githubusercontent.com/542335/132070992-4e9ba64f-a8f4-4459-9102-95684de2cda7.png">

Reply:
Amazing! 🚢